src/lib/themes.js

```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
   }
 
   if (images.headerURL) {
-    const background = normalizeThemeBackground(images.headerURL);
+  // Fx69 update - deprecate headerURL
```

Comment:
small nit: strange indentation
```suggestion
    // Fx69 update - deprecate headerURL
```